po/glossary/nl.po

```diff
@@ -519,16 +517,12 @@ msgid "Mortgage"
 msgstr "Hypotheek"
 
 #. "Adjustable Rate Mortgage"
-#, fuzzy
-#| msgid "Mortgage"
 msgid "mortgage: ARM"
-msgstr "Hypotheek"
+msgstr "Hypotheek: ARM"
```

Comment:
ARM and FRM are not  Dutch terms. Please find translations for those instead. There are  ["Vaste Rentevoet" and "Variabele Rentevoet"](https://www.hypotheek.be/nl/blog/kiezen-voor-vaste-of-variabele-rentevoet,) but I don't know what would be a good shorthand for those.

Reply:
> I don't know what would be a good shorthand for those.

There is no need to have a shorthand.
The problem is: The abbreviations are used in the english sources.
I thougth I would better put them together in the glossary.
There are 3 english abbreviations in 2 different dimensions:
fix or variable rate,
nominal and effective rate/annum.

BTW, I doubt the commiter (weblate bot) will apply changes and I don't know, if he will inform the translator. But some translators have Github accounts and can be notified. In other cases better comment on it or fix it inweblate.

Reply:
Let's remove the abbreviations and say the equivalent in words, so instead of Mortgage: ARM, Adjustable Rate Mortgage... and maybe Fixed/Adjustable Rate Loan would be even better. Similar for Nominal, Annualized, and Effective Interest Rate instead of the abbreviations for those.

Reply:
> Let's remove the abbreviations and say the equivalent in words, so instead of Mortgage: ARM, Adjustable Rate Mortgage... and maybe Fixed/Adjustable Rate Loan would be even better. Similar for Nominal, Annualized, and Effective Interest Rate instead of the abbreviations for those.

@Bob-IT APR and ARM are used in the lists of gnucash/gtkbuilder/assistant-loan.glade. Any idea to get it more translator friendly?
Perhaps replace the ARM list by:
`The rate is fixed for n years, where 0 means unlimited.`